src/bootstrap/llvm.rs

```diff
@@ -380,7 +380,10 @@ impl Step for Llvm {
             cfg.define("LLVM_LINK_LLVM_DYLIB", "ON");
         }
 
-        if target.starts_with("riscv") && !target.contains("freebsd") && !target.contains("openbsd")
+        if target.starts_with("riscv") 
+            && !target.contains("freebsd") 
+            && !target.contains("openbsd") 
+            && !target.contains("netbsd")
```

Comment:
There's some trailing whitespace here. Please run `x fmt` to fix it

Reply:
Thanks.  I tried re-formatting but could not figure out what the automatic checks tried to tell me.  Will look closer & possibly commit & push.